src/Turing.jl

```diff
@@ -82,6 +82,7 @@ using StatsBase: predict
 # Turing essentials - modelling macros and inference algorithms
 export  @model,                 # modelling
         @varname,
+        @submodel,
```

Comment:
Is this intended? I'm fine with it but it seems a bit unrelated here :slightly_smiling_face:

Reply:
Yeah I saw someone on Slack noted that it wasn't exported. I think it's fine to bundle in here.

Reply:
Yep:) From above:

> This requires upping the compat-entry for DPPL, and so I figured we might as well also add re-export of @submodel in this PR.

Reply:
Ah missed your comment :+1: